deploy/staging/deployment.yaml

```diff
@@ -0,0 +1,589 @@
+                Woodhill,
+                Wormwood Scrubs
+            - name: PUBLIC_PRISONS_WITH_SLOT_AVAILABILITY
+              value: >
+                Askham Grange,
+                Aylesbury,
+                Bedford,
+                Buckley Hall,
+                Bure,
+                Cardiff,
+                Channings Wood,
+                Chelmsford,
+                Coldingley,
+                Cookham Wood,
+                Dartmoor,
+                Deerbolt,
+                Downview,
+                Drake Hall,
+                Durham,
+                East Sutton Park,
+                Eastwood Park,
+                Elmley,
+                Erlestoke,
+                Exeter,
+                Featherstone,
+                Feltham (Young People 15-18 only),
+                Feltham (Young Adults 18-21 only),
+                Ford,
+                Foston Hall,
+                Frankland,
+                Full Sutton,
+                Garth,
+                Gartree,
+                Grendon,
+                Guys Marsh,
+                Hatfield Open,
+                Haverigg,
+                Hewell,
+                High Down,
+                Highpoint South,
+                Hindley,
+                Hollesley Bay,
+                Holme House,
+                Hull,
+                Humber,
+                Kirkham,
+                Lancaster Farms,
+                Leicester,
+                Leyhill,
+                Lincoln,
+                Lindholme,
+                Liverpool Social Visits,
+                Low Newton,
+                Maidstone,
+                Medway Secure Training Centre,
+                Moorland Closed,
+                Nottingham,
+                Pentonville,
+                Ranby,
+                Send,
+                Stocken,
+                Stoke Heath,
+                Styal,
+                Sudbury,
+                Swaleside,
+                Usk,
+                Wakefield,
+                Wandsworth,
+                Warren Hill,
+                Wayland,
+                Wealstun,
+                Werrington,
+                Whitemoor,
+                Winchester (Convicted only),
+                Wormwood Scrubs
+            - name: SMTP_DOMAIN
+              value: "mailtrap.io"
+            - name: SMTP_HOSTNAME
+              value: "smtp.mailtrap.io"
+            - name: SMTP_PORT
+              value: "465"
+            - name: SMTP_USERNAME
+              valueFrom:
+                secretKeyRef:
+                  name: prison-visits-booking-staff-secrets
+                  key: smtp_username
+            - name: SMTP_PASSWORD
+              valueFrom:
+                secretKeyRef:
+                  name: prison-visits-booking-staff-secrets
+                  key: smtp_password
+            - name: REMOVE_LOAD_TEST_DATA
+              value: "true"
+            - name: PUBLIC_SERVICE_URL
+              value: "https://public-staging-pvb.dsd.io"
```

Comment:
Do we want to add the new k8s public URL here?

Reply:
Done